entity-management-web/src/main/java/eu/europeana/entitymanagement/web/service/impl/EntityObjectFactory.java

```diff
@@ -77,24 +77,22 @@
 
 	return ret;
     }
-
-    public static Entity createEntityObjectFromCopy(Entity copy) throws EntityCreationException {
-	return createEntityObject(copy.getType());
-//    	switch (EntityTypes.valueOf(copy.getType())) {
-//    	case Organization:
-//    	    return new OrganizationImpl((Organization)copy);
-//    	case Concept:
-//    	    return new ConceptImpl((Concept)copy);
-//    	case Agent:
-//    	    return new AgentImpl((Agent)copy);
-//    	case Place:
-//    	    return new PlaceImpl((Place)copy);
-//    	case Timespan:
-//    	    return new TimespanImpl((Timespan)copy);
-//    	default:
-//    	    throw new RuntimeException("The given type is not supported by the web model");
-//    	}
-    	
+    
+    public static Entity createEntityObjectFromCopy(Entity copy) {
```

Comment:
please remove this method and replace it by createEntityObject(copy.getType()). This implementation is duplicating the code from the existing method

Reply:
Removed completelly.